### PR TITLE
Updates to style for community growth metrics [WIP]

### PR DESCRIPTION
**Pull request by JamesKunstle**
Jupyter Book demo requires some consistency in style- updates have been made to this documents in order to unify the total style of the book.

Signed-off-by: JamesKunstle <jkunstle@bu.edu>

### CODE REVIEW

**Comment by JamesKunstle** [comment]
I'm just pushing this so progress is visible. 

I'd like to be able to stage these changes and see them as a deployment but I'm not sure how that works.
I wasn't available when we moved to JupyterBook so I'm not yet familiar with the process.
@cdolfi 
@isabelizimm

**Comment by isabelizimm** [comment]
Hi James--
to view your changes in a Jupyter Book locally, you can follow [these steps as written by the AICoE,](https://docs.google.com/document/d/1fkLko3tG4Na_3UbGA-YzTqU-H42r-cU-skX07luyXho/edit?usp=sharing) or, these are the same steps, but from the [official documentation](https://jupyterbook.org/start/publish.html#publish-your-book-online-with-github-pages). I believe the `This branch has not been deployed` message is Jupyter Book related. (As a side note, I don't love that wording since _deployment_ can be a loaded term, but that's a setting on the repo I think I can change. :) ) 

When it comes to making a PR, for tracking purposes, it would be best practice to only solve one issue per PR, so we would merge this PR and then make a 2nd one for updating the Jupyter Book deployment.

File: notebooks/community_metrics.ipynb [hunk 1]
```diff
@@ -21,7 +21,15 @@
    "id": "bb5fd073-06f8-4e09-9218-2039ef874104",
```
Comment (cdolfi) [inline notebooks/community_metrics.ipynb:21]: <p>Would probably exclude this/see if there is steps that can work for the rhods and operate first environment as we will be fully moving over soon </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sandiego-rh/sandiego/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by cdolfi** [comment]
The titles look consistent. I have a feeling the organization/titling might change a bit once the other graphs are added in but looking good with the current state!

**Pull request merged**